src/lwc/rd2ElevateInformation/rd2ElevateInformation.js

```diff
@@ -90,38 +88,40 @@ export default class rd2ElevateInformation extends LightningElement {
      * @description Initializes the component with data
      */
     connectedCallback() {
-        getData({ recordId: this.recordId })
-            .then(response => {
-                this.isElevateCustomer = response.isElevateCustomer;
-                this.permissions.alert = response.alert;
-
-                this.permissions.hasAccess = this.isElevateCustomer === true
-                    && response.hasFieldPermissions === true
-                    && isNull(this.permissions.alert);
-
-                if (this.isElevateCustomer === true) {
-                    if (!isNull(this.permissions.alert)) {
-                        this.handleError({
-                            detail: this.permissions.alert
-                        });
-
-                    } else if (response.hasFieldPermissions === false) {
-                        this.handleError({
-                            header: this.labels.flsErrorHeader,
-                            detail: this.labels.flsErrorDetail
-                        });
-
-                    } else if (!isNull(response.errorMessage)) {
-                        this.setErrorStatus(response.errorMessage);
+        if (this.recordId) {
```

Comment:
@ravensisco , in addition to running jest tests, please verify Elevate Information happy path continues to work.